src/Supportive/Console/Command/AnalyseCommand.php

```diff
@@ -26,7 +26,7 @@ class AnalyseCommand extends Command
     final public const OPTION_FAIL_ON_UNCOVERED = 'fail-on-uncovered';
     final public const OPTION_REPORT_SKIPPED = 'report-skipped';
 
-    public static $defaultName = 'analyse|analyze';
+    public static $defaultName = 'analyse';
```

Comment:
I would like to keep the US English spelling, @dbrumann what to you think?

Reply:
analyse is the spelling used in every other place across the whole repository. It was only kept here for Backwards Compatibility.